mytoyota/controller.py

```diff
@@ -26,17 +23,14 @@
     ToyotaLoginError,
 )
 from mytoyota.utils.logs import censor_dict
-from mytoyota.utils.token import is_valid_token
 
+pp = pprint.PrettyPrinter(indent=4)
```

Comment:
This can be removed.
You dont use the PrettyPrinter instantiation anywhere in the code.